### ISSUE TITLE
Allow color picker enter value manually in properties panel

### ISSUE THREAD

**Issue by Insality** [defold/defold#9283]
**Is your feature request related to a problem? Please describe (REQUIRED):**
Color picker is frustrating, we already has several issues relative: https://github.com/defold/defold/issues/6693

But as a most time saving thing, can we enable color raw edit right in the properties panel, without opening a color picker?
Currently to enter a HEX value, I need to make 7 precision clicks

**Describe the solution you'd like (REQUIRED):**
Click on the color name I should to enter a HEX value directly (ex: FFF, F0F0F0, #ABABAB), click on the arrow on the right will open a color picker itself

**Additional context (OPTIONAL):**

https://github.com/user-attachments/assets/b1a730cf-e22d-4031-8bd7-8662d6b218b8

**Comment by d954mas** [comment]
Up, using color picker is annoying and frustrating😭

**Comment by maltsevda** [comment]
Also missing is the ability to pick the color of any pixel in the editor or on-screen - like the eyedropper in Photoshop.

**Comment by Insality** [comment]
> Also missing is the ability to pick the color of any pixel in the editor or on-screen - like the eyedropper in Photoshop.

At least this can be solved by external picking color tool, which can copy text as a HEX (MacOS have a `Digital Colour Meter` for this)

**Comment by itmurker** [comment]
please prioritize this feature🥹🥹🥹

having the option to edit hex color values directly in the properties panel would be a huge time-saver. it feels cumbersome having to go through several clicks just to input a simple color code.

**Comment by Al-V-U** [comment]
I agree, the current color selection is inconvenient, please make it like in Figma. Also, save works only on the current collection, and if you need to transfer between collections, the saved colors disappear!
